testfiles/TF14/14.1-all-pass-1.html

```diff
@@ -0,0 +1,71 @@
+
+<!DOCTYPE html>
+<html lang="en" >
+
+
+<head>
+  <meta charset="UTF-8">
+  <title>LinksInList</title>
+  
+<link rel='stylesheet' href='https://www.w3schools.com/w3css/4/w3.css'>
+
+ 
+
+<style>.trip {
+  list-style-type: none;
+}
+
+.deets {
+  list-style-type: disc;
+}
+
+.triplist {
+  width: 250px;
+}</style>
+
+</head>
+
+
+
+<body>
+
+
+  <div class="w3-container" id="tc_code> 
+<h2 class="w3-text-indigo">Upcoming Trips</h2>
+<ul class="triplist w3-container w3-panel w3-card-4 w3-indigo">
+  <li class="trip">
+    Lake Tahoe
+    <ul class="deets">
+      <li class="w3-hover-red"><a href="#">Details</a></li>
+      <li class="w3-hover-red"><a href="#">Book my trip</a></li>
+    </ul>
+  </li>
+  <li class="trip">

```

Comment:
Suggest only keeping the first bullet and list links. There are multiple links with the same name so I'd remove them to make it a clearer Passing test file.